### DISCUSSION TITLE
Node no longer routing transactions

### DISCUSSION THREAD

**Discussion by von-Hodl** [Ride-The-Lightning/RTL#797] [Q&A] [answered]
Hello, this is my first Github interaction. 

My node that used route transactions daily has suddenly stopped. 0 transactions for a month, not 1.

My channels are balanced my fees are low. (base fee .036 ppm 350)
I can open and close channels
I can make payments I can receive payments.

I don't know what to look at to try and figure out why I'm suddenly a ghost.

**Comment by saubyk** [comment, marked as answer]
Hi @von-Hodl welcome to the RTL discussions forum. :-)

There can be variety of factors for low routing traffic volume and a lot of those might not be under your control. Some of it can be:
- Low on-chain transaction volume, resulting in lower lightning transactions overall
- Low business activity on the nodes you are routing to
- Increased routing competition

Follow these basics to maintain good routing probability via your node:
- High uptime (node should not be offline for long periods)
- Reasonably balanced channels (maintain a good remote/local channel balance ratio)
- Reasonable routing fee policy on your channels (jacking up the routing fee too much can result in low traffic)
- Monitor your peers for inactive channels. Peers which go offline too often render the channels inactive for routing.

If you've ensured all of the above, you can try playing with the routing fee policy to see if that has an impact on the volume.

In your specific case, I would recommend increase the base fee a bit (e.g. 3000 msats) and reduce the ppm (e.g. 50) and see if that makes any difference. Do keep in mind that you should give your node a few days after adjusting the fee rate to see how it plays out.

Hope this helps. :-)

**Reply by von-Hodl** [reply]
ok just lowered the fees yesterday. I'll give it until the weekend and see what happens. If i continue to get nothing. Then I'll try the fees you suggested.

Thank you

**Reply by von-Hodl** [reply]
Ok no change. I've updated my fees based on your suggestion. We'll see what happens.

**Reply by von-Hodl** [reply]
That worked. 6 forwards just routed.

Thank you for your time.
